cocos/tween/actions/action.ts

```diff
@@ -207,112 +208,3 @@ export abstract class FiniteTimeAction extends Action {
 
     abstract reverse (): FiniteTimeAction;
 }
-
-/*
- * Changes the speed of an action, making it take longer (speed > 1)
- * or less (speed < 1) time. <br/>
- * Useful to simulate 'slow motion' or 'fast forward' effect.
- */
-export class Speed extends Action {
```

Comment:
Why delete it?

Reply:
It's unused code.

Reply:
Is it exported?

Reply:
No, all actions tween module use are internal.